modules/graceful-restart.adoc

```diff
@@ -119,14 +119,14 @@ Check that there are no cluster Operators with the `DEGRADED` condition set to `
 [source,terminal]
 ----
 NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
-authentication                             4.9.0     True        False         False      59m
-cloud-credential                           4.9.0     True        False         False      85m
-cluster-autoscaler                         4.9.0     True        False         False      73m
-config-operator                            4.9.0     True        False         False      73m
-console                                    4.9.0     True        False         False      62m
-csi-snapshot-controller                    4.9.0     True        False         False      66m
-dns                                        4.9.0     True        False         False      76m
-etcd                                       4.9.0     True        False         False      76m
+authentication                             4.10.0     True        False         False      59m
```

Comment:
You might want to delete a space after the 4.10 - since it's adding an extra character, it's throwing off the following columns from being aligned properly

Can you also check any of the other modules in the PR that have columns that would also be affected by this?

Reply:
@bergerhoffer Good catch!